shell/app-shell/elements/arc-steps.js

```diff
@@ -77,9 +77,7 @@ class ArcSteps extends Xen.Base {
       // do something smarter than literal matching anyway...
       // Find all mapped handles to be remembered.
       // Store as string, as we'll only use it to find exact matches later. (String is easier to compare)
-      // TODO(wkorman): Rename `views` below to `handles` which may
-      // necessitate revising the launcher.
```

Comment:
I wonder what we had in mind here re: the launcher, but if the Selenium demo flow tests pass then we should be ok.

Reply:
selenium test passes and local app-shell works too, so hopefully it is ok.